contracts/src/v0.8/dev/ocr2dr/OCR2DROracle.sol

```diff
@@ -148,6 +149,8 @@ contract OCR2DROracle is OCR2DROracleInterface, OCR2Base, AuthorizedOriginReceiv
       revert ReportInvalid();
     }
 
+    // Instead of a variable cost for users based on how many fulfillments are batched together, can we instead set a reasonable constant value to account for the gas used for validation?
```

Comment:
If we expect that the number of reports will be fairly consistent, then yes this could work.
But again this is such a tiny amount of gas saved avoiding one line.